examples/generated-src/wasm/NativeSortOrder.cpp

```diff
@@ -13,16 +13,17 @@ namespace {
             DESCENDING : 1,
             RANDOM : 2,
         }
+        ;
     })
 }
 
-void NativeSortOrder::staticInitialize() {
+void NativeSortOrder::staticInitializeConstants() {
```

Comment:
It's related. I have to separate the initializing of namespace aliasing, which now taking the `staticInitialize` name and initializing of constants, because they can independently exist.